src/libraries/System.Text.RegularExpressions/tests/UnitTests/Stubs.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Buffers;
+using System.Collections;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace System.Text.RegularExpressions
+{
+    internal sealed class RegexReplacement
```

Comment:
instead of creating this file, can you break out this class from the existing Stubs.cs to avoid duplication?

Reply:
You mean the one in the generator? Is it really worth creating a common file for a handful of stubbed out members?

Reply:
Your choice, I think generally we should have a good reason to duplicate code.

Reply:
The reason here is the code is boilerplate (there's literally no logic in it, just the stubs) and it's mote boilerplate to avoid it than not.

Reply:
fair enough